spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/client/TraceWebClientBeanPostProcessor.java

```diff
@@ -277,6 +280,14 @@ public Context currentContext() {
 			return this.context;
 		}
 
+		@Override
+		public Object scanUnsafe(Scannable.Attr key) {
+			if (key == Scannable.Attr.RUN_STYLE) {
+				return Scannable.Attr.RunStyle.SYNC;
+			}
+			return null;
```

Comment:
maybe trailing comment why null or add Nullable back to the override for clarity. That way it is easy to know if this is a bug or not even if less familiar